p2p/net/connmgr/decay.go

```diff
@@ -163,7 +164,8 @@ func (d *decayer) process() {
 	for {
 		select {
 		case now = <-ticker.C:
-			d.lastTick.Store(now)
+			nn := now
+			d.lastTick.Store(&nn)
```

Comment:
Why not `&now`?

Reply:
now is declared outside the block and reassigned at line 219. storing a pointer to now would cause it to use this value of 219 sometimes.

Reply:
Good catch!